scripts/fill_dcp1_file_metadata/fill_dcp1_metadata.py

```diff
@@ -0,0 +1,188 @@
+import requests as rq
+import argparse
+import json
+import logging
+
+from requests.exceptions import HTTPError
+
+from hca_ingest.api.ingestapi import IngestApi
+
+"""
+The script follows the next steps to fix the missing size and fileContentType for DCP1 datasets:
+
+1. Check the project UUID provided is valid 
+2. Retrieve all data file metadata from azul
+3. Parse and massage the data to avoid DCP/1 / HCA Release / DCP/2 analysis files (Not in ingest)
+4. For each of the metadata files:
+   a. Check metadata is not already filled (Priority: Ingest > Azul)
+   b. PUT metadata file
+"""
+INGEST_TOKEN = ""
+
+# TODO: Check if file metadata is editable (submission editable). Don't know how to do it without 1 API call/file
+
+
+class NotPublishedUuid(HTTPError):
+    def __init__(self, response: rq.Response, uuid: str):
+        super().__init__(f"UUID {uuid} not found in Azul: GET request errored with status code {response.status_code}")
+
+
+def is_published_uuid(uuid: str) -> str:
+    """
+    Check if the UUID provided corresponds to a published Azul project
+    :param uuid:    UUID of the project which contains the files
+    :return uuid:
+    """
+    r = rq.get(f"https://service.azul.data.humancellatlas.org/index/projects/{uuid}")
+    try:
+        r.raise_for_status()
+    except HTTPError:
+        raise NotPublishedUuid(r, uuid)
+    return uuid
```

Comment:
This is a bad coding practice.
Why to you return the given uuid?
Functions named `is_xyz` are boolean predicates by convention. It should return true or false.
You can accept to http statuses from the http call: 200 if the uuid is found, 404 if not, and return `True` for the former case `False` for the latter. If a different status is returned, you can use `raise_for_status`.

Reply:
I would suggest [response.ok](https://requests.readthedocs.io/en/latest/api/#requests.Response.ok):
```
r = rq.get(...)
return r.ok
```